v2/slogger/rolling_file_appender/hidden_windows.go

```diff
@@ -0,0 +1,27 @@
+// +build windows
+
+package rolling_file_appender
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+func createHidden(name string) (*os.File, error) {
+	var sa syscall.SecurityAttributes
+	sa.Length = uint32(unsafe.Sizeof(sa))
+	sa.InheritHandle = 1
+	handle, err := syscall.CreateFile(
+		syscall.StringToUTF16Ptr(name),
+		syscall.GENERIC_READ|syscall.GENERIC_WRITE,
+		syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE,
+		&sa,
+		syscall.CREATE_ALWAYS,
+		syscall.FILE_ATTRIBUTE_HIDDEN,
+		0)
```

Comment:
@markbenvenuto I selected parameter values here to match what's used by the Go runtime when you call `os.Create`, even though we probably don't need `FILE_SHARE_WRITE` and `InheritHandle` for what we're doing.